polaris-react/src/components/Box/Box.tsx

```diff
@@ -132,6 +132,8 @@ export interface BoxProps extends React.AriaAttributes {
   outlineStyle?: LineStyles;
   /** Outline width */
   outlineWidth?: BorderWidthScale;
+  /** Adds display: flex */
+  flex?: boolean;
```

Comment:
I'm not too sure about the API design for the Box component but this almost feels like we should have this prop be `display: 'flex' | 'inline-block'| etc...` and have a `flex` prop that we could use for defining things like `1 0 auto`. Granted, that could also be expressed as `flexGrow` and it's other attributes. What do you think?

Reply:
Yea actually I am ok with that.